logging/impl_test.go

```diff
@@ -304,6 +315,13 @@ func TestSublogger(t *testing.T) {
 }
 
 func TestLoggingWithFields(t *testing.T) {
+	// Disable log deduplication for this test, as it logs "noisily" and makes
+	// assertions on those logs.
+	DisableLogDeduplication.Store(true)
```

Comment:
I think* this could global variable could instead, without loss of generality, be put on the logger registry object? "Registry" certainly doesn't sound like the right name, but I think it's global enough for our purposes. And it would isolate the value for tests.